scripts/_lib.sh

```diff
@@ -1,4 +1,30 @@
-npmCommand="yarn"
+npmCommand="npm"
+babelCliVersion="^6.18.0"
+
+function initialize {
+  path=${1-.}
+
+  # Install yarn to Meteor globals. Enable `meteor yarn` subcommand.
+  if [ ! $(meteor npm list -gs --depth=0 | grep -qo "yarn") ]; then
+    printf "${BLUE}Installing yarn to Meteor globals...\n${NC}"
+    meteor npm install yarn -g
+  fi
+
+  # Install babel-cli for running ES6 scripts
+  if [ ! -d "$path/node_modules/babel-cli" ]; then
+    printf "${BLUE}Installing babel-cli...\n${NC}"
+    npm install babel-cli@$babelCliVersion --save-dev
```

Comment:
In fact, I removed that. The idea is going to be to include this library here and always expose the commands from `-scripts` as a `.sh` via `bin` config. But I'll do that when moving all our babel scripts here...